src/app/components/ManualEntry.svelte

```diff
@@ -0,0 +1,133 @@
+<script lang="ts">
+  /**
+   * Handles the click when the delte button is pressed
+   * TODO
+   */
+  const handleDelete = (): void => {};
+</script>
+
+<style>
+  .language__sources-manual-entry {
+    border-radius: 20px;
+    padding: 5px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .language__sources-manual-entry-tablename input {
+    height: 30px;
+    width: 400px;
```

Comment:
Please follow the guidelines set here: https://github.com/eddieantonio/predictive-text-studio/blob/production/docs/conventions.md#css-prefer-rem-to-px-for-positioning and here: https://github.com/eddieantonio/predictive-text-studio/blob/production/docs/conventions.md#css-prefer-em-to-px-for-font-size

Please do not copy-paste directly from Figma! But definitely use Figma as a guide 😅